simulator-core/src/test/java/com/evbox/everon/ocpp/it/security/BasicAuthenticationIt.java

```diff
@@ -67,7 +68,7 @@ void expectSuccessfulAuth() {
 
 
         await().untilAsserted(() -> {
-            Map<String, String> receivedCredentials = ocppMockServer.getReceivedCredentials();
+            Map<String, String> receivedCredentials = new HashMap<>(ocppMockServer.getReceivedCredentials());
```

Comment:
i think it's better to make a copy when doing ocppMockServer.getReceivedCredentials so that other tests know to do it as well..

also don't forget to put volatile in front of the map declaration.

Reply:
Done